build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -21,14 +21,20 @@
 TOOL_PATH=${TOP_PATH}/build/tools/amebad
 IMG_TOOL_PATH=${TOOL_PATH}/image_tool
 
+OS_PATH=${TOP_PATH}/os
+CONFIG=${OS_PATH}/.config
+source ${CONFIG}
+
 function pre_download()
 {
 	source ${TOP_PATH}/os/.bininfo
 	cp -p ${BIN_PATH}/km0_boot_all.bin ${IMG_TOOL_PATH}/km0_boot_all.bin
 	cp -p ${BIN_PATH}/km4_boot_all.bin ${IMG_TOOL_PATH}/km4_boot_all.bin
 	cp -p ${BIN_PATH}/${KERNEL_BIN_NAME} ${IMG_TOOL_PATH}/${KERNEL_BIN_NAME}
-	if test -f "${SMARTFS_BIN_PATH}"; then
-		cp -p ${BIN_PATH}/rtl8721csm_smartfs.bin ${IMG_TOOL_PATH}/rtl8721csm_smartfs.bin
+	cp -p ${BIN_PATH}/micom ${IMG_TOOL_PATH}/micom
+	cp -p ${BIN_PATH}/wifi ${IMG_TOOL_PATH}/wifi
+	if test -f ${BIN_PATH}/common_binary; then
```

Comment:
You missed to change the common binary name based on config.
Please check this.

Reply:
I updated missed changes. Thank you!